libraries/chain/db_debug.cpp

```diff
@@ -96,7 +96,7 @@ void database::debug_dump()
    if( total_balances[asset_id_type()].value != core_asset_data.current_supply.value )
    {
       edump( (total_balances[asset_id_type()].value)(core_asset_data.current_supply.value ));
-      exit(0);
+      throw;
```

Comment:
Uh, at least use FC_THROW with an explanatory message. Perhaps I should have been more explicit. :-)

Reply:
sorry, changing now :)